test/visibility/only/onlyExceptAllows.chpl

```diff
@@ -1,7 +1,7 @@
 // Verifies the behavior of use chains that start with an 'only' and have
 // an 'except' in them
 module Outer {
```

Comment:
I think this this is the last of the use tests with comments that need to be updated